src/main/resources/static/js/common.js

```diff
@@ -308,15 +308,18 @@ function executeIfLdapIsDeactivated() {
     $("#ldapUserName").text("");
 }
 
-function downloadSubscriptions(subscriptionData, fileName) {
+function downloadSubscriptions(subscriptionData, subscriptionNames) {
     var foundSubscriptions = subscriptionData.foundSubscriptions;
     var jsonData = JSON.stringify(foundSubscriptions, null, 2);
-    if (jsonData == undefined) {
+    if (jsonData == undefined || subscriptionNames == undefined) {
         logMessage("Subscription data to download was not found");
         return;
     }
-    if (fileName == undefined) {
-        fileName = "subscriptionsData.json";
+    var fileName = ".json";
+    if (stringContainsSubstring(subscriptionNames, ",")) {
+        fileName = "subscriptionsData" + fileName;
+    } else {
+        fileName = subscriptionNames + fileName;
```

Comment:
Is there a limit to how long the filename is allowed to be? What happens if I have a 20 subscriptions with long names which I download all at once...? Perhaps keep the generic subscriptionsData.json for when downloading multiple subscriptions?

Reply:
Skimmed through a bit too fast before :) Looks good now, I'll approve!